### PR TITLE
[stable/sumologic-fluentd] Fix sourceCategoryPrefix default value

### LINKED ISSUE #10020: README update - sourceCategoryPrefix default value
**Is this a BUG REPORT or FEATURE REQUEST?** (choose one):
BUG REPORT (documentation)

**Version of Helm and Kubernetes**:
N/A

**Which chart**:
sumologic-fluentd

**What happened**:
The default value for `sumologic.sourceCategoryPrefix` in the README file is `Nil` but it should be `kubernetes/` according to [the sumologic documentation](https://github.com/SumoLogic/fluentd-kubernetes-sumologic).

The comment in the `values.yaml` file also requires an update to reflect the same thing.

**What you expected to happen**:
See the correct value in the documentation and in the default `values.yaml` file.

**How to reproduce it** (as minimally and precisely as possible):
N/A

**Anything else we need to know**:
N/A

### PR DESCRIPTION
**Pull request by Dean-Coakley**
#### What this PR does / why we need it:
* Make chart follow sourceCategoryPrefix default value: https://github.com/SumoLogic/fluentd-kubernetes-sumologic#environment-variables

#### Which issue this PR fixes
- fixes: https://github.com/helm/charts/issues/10020

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Comment by Dean-Coakley** [comment]
@seanknox @lachie83 @davidkarlsen Can I get a quick review on this? Thanks

File: stable/sumologic-fluentd/README.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ The following table lists the configurable parameters of the sumologic-fluentd c
 | `sumologic.sourceName` | Set the sumo `_sourceName` | `%{namespace}.%{pod}.%{container}` |
 | `sumologic.sourceHost` | Set the sumo `_sourceHost` | `Nil` |
 | `sumologic.sourceCategory` | Set the sumo `_sourceCategory` | `%{namespace}/%{pod_name}` |
-| `sumologic.sourceCategoryPrefix` | Define a prefix, for `_sourceCategory` | `Nil` |
+| `sumologic.sourceCategoryPrefix` | Define a prefix, for `_sourceCategory` | `/kubernetes` |
```
Comment (darend) [inline stable/sumologic-fluentd/README.md:74]: Existing deployments may be relying on this behavior. I recommend a minor version bump to reflect this

Comment (Dean-Coakley) [reply stable/sumologic-fluentd/README.md:74]: That makes sense. Pushing now

**Comment by darend** [comment]
Sorry @Dean-Coakley , you will also need to rebase

**Comment by Dean-Coakley** [comment]
Hm I didn't see any conflicts earlier. Should be good now @darend

**Comment by darend** [comment]
@frankreno this looks good to me, can you do the final approval since it impacts existing deployments

**Comment by Dean-Coakley** [comment]
@frankreno Can you please review these changes?

**Comment by frankreno** [comment]
@Dean-Coakley : Thanks for the PR. sorry for the delay in reviewing this.  So a concern I have is that today, we have customers who use helm, and those who do not.  For those customers who do not, we define a YAML and specify some default values in the Docker File. In the Docker Image, we define multiple default values. For example, the property this is for is defined here:

https://github.com/SumoLogic/fluentd-kubernetes-sumologic/blob/master/Dockerfile#L39

So the values.yaml is set to Nil because the docker image has it set as an environment variable.  This PR only addresses one case of this.  To be consistent we should do all, but I am wondering if a simple comment with a pointer to the Docker File and what default environment variable serves it would suffice.

Thoughts?

Another thing I noticed is you changed the value to `/kubernetes`.  With this value, your source categories might look off without the ending `/`.

**Pull request closed without merging**